requirements-dev.in

```diff
@@ -0,0 +1,3 @@
+-c requirements.txt
+django-debug-toolbar
+pydevd-pycharm==242.23726.102  # pinned to appropriate version for current pycharm
```

Comment:
I can see the argument for pinning this. I can also see an argument for leaving it unconstrained. Either way requires action when any team member upgrades.

Reply:
I'd prefer to not pin it, but it's required that it's pinned at the moment. 

The issue is pycharm is very picky about the version of pydevd-pycharm it'll work with at any point in time, and as a good example of this, when I first set this up there is a newer version of pydevd-pycharm, but it's not compatible with the current version of pycharm and pycharm will complain if you use it... 

I see now they release another version 2 hours ago so I'll test if that works soon but for now this is compatible.